ToolBox/CoinApiDataConverter/CoinApiDataConverter.cs

```diff
@@ -158,6 +158,14 @@ private void ProcessEntry(CoinApiDataReader coinapiDataReader, FileInfo file)
 
             foreach (var tick in ticks)
             {
+                if (tick.Value == -1 || tick.BidPrice == -1 || tick.AskPrice == -1)
```

Comment:
Don't think `tick.Value == -1` will happen? Since it's
```
            Value = BidPrice + AskPrice;
            if (BidPrice * AskPrice != 0)
            {
                Value /= 2m;
            }
```

Reply:
I believe `Tick.SetValue()` is never called on the `Tick` we construct inside the `CoinApiDataReader` since we construct it explicitly and not w/ a Reader or constructor method.

Also, I think the math for this logic resolves to:

```c#
Value = -2 == (-1 + -1);
if (BidPrice * AskPrice == (-1 * -1) == 1; 1 != 0) 
{
    Value /= 2m;
    Value == -1m; // true
}
```

Also, as per your suggestion, will be flagging the Ticks as `Tick.Suspicious = true` before merge :+1: